docs/querying/dart.md

```diff
@@ -0,0 +1,97 @@
+---
+id: dart
+title: "SQL queries using the Dart query engine"
+sidebar_label: "Dart query engine"
+description: Use the Dart query engine for light-weight queries that don't need all the capabilities of the MSQ task engine.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info[Experimental]
+
+Dart is experimental. For production use, we recommend using the other available query engines.
+
+:::
+
+
+Use the Dart query engine for light-weight queries that don't need all the capabilities of the MSQ task engine. For example, use Dart for GROUP BY queries that have intermediate results consisting of hundreds of millions of rows. In this case, the Dart engine's multi-threaded workers perform in-memory shuffles using locally cached data without pulling from deep storage.
+
+You can query batch or realtime datasources with Dart.
```

Comment:
One difference that should be documented, Dart queries realtime by default, while MSQ task doesn't. If you don't want to query realtime sources, you have to set the parameter accordingly.

Reply:
What's the parameter?